src/data/symbol/symbol.tsv

```diff
@@ -125,7 +126,9 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	°	ど たんい きごう	度		SYMBOL
 記号	′	’ ふん たんい きごう	分		SYMBOL	PRIME
 記号	″	” びょう たんい きごう	秒		SYMBOL	DOUBLE PRIME
-記号	℃	ど たんい せっし どC どしー C	単位		OTHER
+記号	℃	ど たんい せっし どC どc どしー C	単位		OTHER
+記号	°F	かし どF どf どえふ F	単位		OTHER
+記号	℉	かし どF どf どえふ F	単位		OTHER
```

Comment:
Since ℉ (U+2109) is a compatibility character, it's not recommended to use it newly.
Let's remove this line.

Reply:
I understand you prefer the ° + F form.
Should we add the ° + C form for ℃, too?

Reply:
It's fine to add "° + C" too. I'd prefer to keep "℃" (U+2103) as it's already widely used.